webapp/lib/controller.dart

```diff
@@ -31,42 +35,37 @@ class UserData extends Data {
     return "UserData($displayName, $email, $photoUrl)";
   }
 }
+class ProjectData extends Data {
+  String projectName;
+  List<String> projectMembers;
 
-class ConfigurationTagData extends Data {
-  String selectedTag;
-  String tagToAdd;
-  ConfigurationTagData({this.selectedTag, this.tagToAdd});
+  ProjectData(this.projectName, this.projectMembers);
 }
 
-class ConfigurationResponseData extends Data {
-  String parentTag;
-  String language;
-  String text;
-  int index;
-  ConfigurationResponseData({this.parentTag, this.index, this.language, this.text});
-}
-
-Map<String, List<List<String>>> configurationTagData;
-Set<String> additionalConfigurationTags;
 List<String> configurationResponseLanguages;
 
 model.User signedInUser;
 
+ProjectData project;
+
 void init() async {
   setupRoutes();
   view.init();
   await platform.init();
 }
 
 void initUI() {
-  router.routeTo(window.location.hash); //TODO This is just temporary initialization becuase we don't have a complete app
-  view.navView.projectTitle = 'COVID IMAQAL'; //TODO To be replaced by data from model
+  if (project == null) {
+    router.routeTo('#/project-selector');
+  }
+  router.routeTo(window.location.hash);
```

Comment:
if I understand correctly, `initUI` is going to get called only once, when the page is loaded, which means that `project` will always be null, and so it will always go to the `#/project-selector` page. This means that this line is never going to get called - is that right? In which case, we could just have
 
```suggestion
  router.routeTo('#/project-selector');
```

I think long term we're going to want to store the project name in the URL, and so on page load we can pick it up from there, but this can come in a future PR.

Reply:
Yeah. I see how it won't be called. Changed